app/views/space_contacts/_space_contact.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= turbo_frame_tag dom_id(space_contact) do %>
+  <div class="mx-6 border-2 border-gray-100 p-2 mt-2 bg-gray-50">
+    <div class="flex flex-cols-2">
+      <h3 class="flex-grow"><%= space_contact.title %></h3>
+      <div class="place-self-end pb-1">
+        <%= link_to edit_space_contact_path(space_contact), class: 'unstyled-link edit-button collapsable' do %>
+          <span class="text">
+            <%= t('space_contacts.edit') %>
+          </span>
+            <%= inline_svg 'edit', alt: t('space_contacts.edit'), title: t('space_contacts.edit') %>
+        <% end %>
+        <%= link_to space_contact_path(space_contact), method: :delete, class: 'unstyled-link edit-button collapsable' do %>
```

Comment:
Would love a confirmation here. Added it to Trello for a future PR, if we can't easily get it here: https://trello.com/c/8qrdTeuB/130-are-you-sure-n%C3%A5r-man-sletter-kontakter-og-andre-ting-lage-en-felles-funksjon-som-alle-slette-knapper-kan-bruke

Reply:
Tried the easy way now, it did not work. Will have to take a slightly deeper look into it. 
Will move the button to inside the edit form to make it "easier to not" press it :D